Formula/dub.rb

```diff
@@ -14,13 +13,7 @@ class Dub < Formula
     sha256 "5bab5095000421e561fbf41268c615e079023b86a0d6d7ed00328ab52a7db906" => :mavericks
   end
 
-  devel do
-    url "https://github.com/dlang/dub.git",
-        :tag => "v1.1.0-beta.1",
-        :revision => "1824d16056a680fcca52d3ce87fb43d3679ff3ee"
-  end
-
-  depends_on "pkg-config" => :build
+  depends_on "pkg-config" => :recommended
```

Comment:
`dub` uses it at runtime if it can find it, but is also perfectly happy without. It's no longer used for building `dub` itself unless you're building it with an old compiler.